src/qml/FeatureForm.qml

```diff
@@ -347,7 +349,10 @@ Page {
       return
     }
 
-    save()
+    if ( ! save() ) {
+      displayToast( qsTr( 'Unable to save changes') )
+      return
```

Comment:
Shouldn't it go back to "Edit" state anyway? I'm unsure.

Reply:
I put it back to keep the previous behavior.